### ISSUE TITLE
Howto Save info Report?

### ISSUE THREAD

**Issue by carlymx** [arunsivaramanneo/GPU-Viewer#53]
Hello.

Is there a way to save the information given by gpu-viewer to a file?

Thank you.

**Comment by arunsivaramanneo** [comment]
Hi , Sorry don't have this facitlity , i am already taking the information from Vulkaninfo , glxinfo , clinfo, vdpauinfo and displaying in the application .. just to understand why do you need to save the information ?

**Comment by carlymx** [comment]
I know that this info comes out of their respective commands.
And I also know that it is possible to pass that info to a file easily through the terminal like for example:

`vulkaninfo > ./vulkan.txt`

The answer to your concern about my need for the ability to export reports arises from the same need to use your program, knowing that the same information can be displayed with a simple command.

In a nutshell. Anyone who uses a program as interesting as the one you are creating may need to export full or partial reports, for example in html. Take it as a humble suggestion to keep in mind for the future.

**Comment by arunsivaramanneo** [comment]
thank you .. i will try to look into it ... i have few things in mind .. but i am not sure if i am able to implement it ... if i get time .. i will check.. thank you for our suggestion

**Comment by carlymx** [comment]
Thank you for your time and for your wonderful program. I do my little things, but not as well as you.

**Comment by arunsivaramanneo** [comment]
As this already a extract of vulkaninfo, glxinfo, clinfo etc .. so don't make sense to create another report
